modular_skyrat/modules/death_consequences_perk/death_consequences_customization.dm

```diff
@@ -0,0 +1,228 @@
+/datum/preference/toggle/expand_death_consequences_options // simple filler option that lets you expand all the below
+	savefile_key = "dc_expand_options"
+
+	category = PREFERENCE_CATEGORY_NON_CONTEXTUAL
+	savefile_identifier = PREFERENCE_CHARACTER
+
+	default_value = FALSE
+
+/datum/preference/toggle/expand_death_consequences_options/is_accessible(datum/preferences/preferences)
+	. = ..()
+	if (!.)
+		return FALSE
+
+	return DEATH_CONSEQUENCES_QUIRK_NAME in preferences.all_quirks
+
+/datum/preference/toggle/expand_death_consequences_options/apply_to_human(mob/living/carbon/human/target, value, datum/preferences/preferences)
+	return FALSE
+
+/datum/preference/toggle/death_consequences/apply_to_human(mob/living/carbon/human/target, value, datum/preferences/preferences)
+	return FALSE
+
+/datum/preference/numeric/death_consequences
+	abstract_type = /datum/preference/numeric/death_consequences
+	category = PREFERENCE_CATEGORY_NON_CONTEXTUAL
+	savefile_identifier = PREFERENCE_CHARACTER
+
+	step = 0.01
+
+/datum/preference/numeric/death_consequences/is_accessible(datum/preferences/preferences)
+	. = ..()
+	if (!.)
+		return FALSE
+
+	if (!(DEATH_CONSEQUENCES_QUIRK_NAME in preferences.all_quirks))
+		return FALSE
+
+	return preferences.read_preference(/datum/preference/toggle/expand_death_consequences_options)
+
+/datum/preference/numeric/death_consequences/apply_to_human(mob/living/carbon/human/target, value, datum/preferences/preferences)
+	return FALSE
+
+/datum/preference/numeric/death_consequences/starting_degradation
+	savefile_key = "dc_starting_degradation"
+
+	minimum = 0
+	maximum = DEATH_CONSEQUENCES_MAXIMUM_THEORETICAL_DEGRADATION
+
+/datum/preference/numeric/death_consequences/starting_degradation/create_default_value()
+	return minimum
+
+/datum/preference/numeric/death_consequences/max_degradation
+	savefile_key = "dc_max_degradation"
+
+	minimum = 0
+	maximum = DEATH_CONSEQUENCES_MAXIMUM_THEORETICAL_DEGRADATION
+
+/datum/preference/numeric/death_consequences/max_degradation/create_default_value()
+	return DEATH_CONSEQUENCES_DEFAULT_MAX_DEGRADATION
+
+/datum/preference/numeric/death_consequences/living_degradation_recovery_per_second
+	savefile_key = "dc_living_degradation_recovery_per_second"
+
+	minimum = -100 // if you want, you can just die slowly
```

Comment:
I'm not sure that's necessarily a good idea, but I also don't know how high of a number you need in order to die.

But considering the size of our rounds on Skyrat, I don't think this is really a good idea, they'll be DNR by round end in any case, so they might as well take the DNR quirk.

Reply:
Especially, I don't want more quirks that force your death after X amount of time, it just tends to make roleplay awkward in most cases, and then require admin intervention.

Reply:
well it doesnt necessarily FORCE their death unless they specifically CHOOSE to have it force their death

they can take rezadone, strange reagent, eigen, sansu, etc. hell they can even lie down on stasis to slow it

Reply:
if its still a "nuh-uh", i can remove it though